src/java.base/share/classes/java/util/Currency.java

```diff
@@ -432,61 +433,78 @@ public static Currency getInstance(Locale locale) {
     }
 
     /**
-     * Gets the set of available currencies.  The returned set of currencies
-     * contains all of the available currencies, which may include currencies
-     * that represent obsolete ISO 4217 codes.  The set can be modified
+     * {@return a set of available currencies} The returned set of currencies
+     * contains all the available currencies, which may include currencies
+     * that represent obsolete ISO 4217 codes. If there is no currency available
+     * in the runtime, the returned set is empty. The set can be modified
```

Comment:
An apiNote might be appropriate to refer/recommend using the new `availableCurrencies()` API.
And/or an \@see link to the method.

Reply:
Thanks Roger, incorporated both your suggestions in https://github.com/openjdk/jdk/pull/23165/commits/dcb4a9bfea1002044483aea483af8e6107eb525d. Note, I intentionally did not add any wording regarding the defensive copy in the `apiNote`, considering that is an implementation specific caveat.